### PR TITLE
Add PR and Issue templates

### PR DESCRIPTION
**Pull request by lnhote**
Copied from tidb repo and changed a little.
see issue: https://github.com/pingcap/parser/issues/26

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ - Integration test
+ - Manual test (add detailed scripts or steps below)
+ - No code
+
+Code changes
+
+ - Has exported function/method change
+ - Has exported variable/fields change
+ - Has interface methods change
+ - Has persistent data change
```
Comment (kennytm) [inline .github/pull_request_template.md]: Is there any "persistent data" for this repository?

Comment (lnhote) [reply .github/pull_request_template.md]: please review again.

**Comment by kennytm** [comment]
PTAL @shenli @tiancaiamao

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### What is changed and how it works?
+
+
+### Check List <!--REMOVE the items that are not applicable-->
+
+Tests <!-- At least one of them must be included. -->
+
+ - Unit test
+ - Integration test
```
[the remark this answers is not in the document]
Comment (kennytm) [reply .github/pull_request_template.md]: Well #155 exists 🤣

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F41B Bug Report"
```
[the remark this answers is not in the document]
Comment (lnhote) [reply .github/ISSUE_TEMPLATE/bug-report.md]: It is an emoji for bug.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+2. What did you expect to see?
+
+
+
+3. What did you see instead?
+
+
+
+4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?
```
[the remark this answers is not in the document]
Comment (lnhote) [reply .github/ISSUE_TEMPLATE/bug-report.md]: How about this: What version of TiDB SQL Parser are you using?

**Comment by tiancaiamao** [comment]
LGTM

**Comment by lnhote** [comment]
Is there something wrong with this pr?

**Comment by shenli** [comment]
@lnhote Thanks for your contribution and sorry for the late reply. 
LGTM

**Pull request merged**